describe.js

```diff
@@ -1,116 +1,115 @@
-'use strict';
+const types = {};
 
-module.exports = describe;
-
-function describe(node) {
-    return types[node.type](node);
-}
+const describe = node => {
+  return types[node.type](node);
+};
 
-var types = {};
+export default describe;
 
-types.text = function text(node) {
-    return node.text;
+types.text = node => {
+  return node.text;
 };
 
-types.echo = function echo(node) {
-    return S(node.expression);
+types.echo = node => {
+  return S(node.expression);
 };
 
-types.opt = function opt(node) {
-    return '(Q ' + node.question.join(' ') + ') (A ' + node.answer.join(' ') + ')';
+types.opt = node => {
+  return '(Q ' + node.question.join(' ') + ') (A ' + node.answer.join(' ') + ')';
 };
 
-types.goto = function goto(_node) {
-    return '';
+types.goto = _node => {
+  return '';
 };
 
-types.call = function call(node) {
-    return node.label + '(' + node.args.map(S).join(' ') + ') esc ' + node.branch;
+types.call = node => {
+  return node.label + '(' + node.args.map(S).join(' ') + ') esc ' + node.branch;
 };
 
-types.def = function def(node) {
-    return '(' + node.locals.join(' ') + ')';
+types.def = node => {
+  return '(' + node.locals.join(' ') + ')';
 };
 
-types.jump = function jump(node) {
-    return node.branch + ' if ' + S(node.condition);
+types.jump = node => {
+  return node.branch + ' if ' + S(node.condition);
 };
 
-types.switch = function _switch(node) {
-    var desc = '';
-    if (node.variable) {
-        desc += '(' + node.variable + '+' +  node.value + ') ' + S(node.expression);
-    } else {
-        desc += S(node.expression);
-    }
-    desc += ' (' + node.branches.join(' ') + ') W(' + node.weights.map(S).join(' ') + ')';
-    return desc;
+types.switch = node => {
+  let desc = '';
+  if (node.variable) {
+    desc += '(' + node.variable + '+' + node.value + ') ' + S(node.expression);
+  } else {
+    desc += S(node.expression);
+  }
+  desc += ' (' + node.branches.join(' ') + ') W(' + node.weights.map(S).join(' ') + ')';
```

Comment:
uplift to backtick wen

Reply:
Adding a commit to adopt the prefer-template and apply the autofix.